.github/workflows/static-checks.yml

```diff
@@ -41,6 +42,32 @@ jobs:
               # See: https://github.com/WordPress/gutenberg/pull/39865
               run: npm install
 
+            - name: Debug github.head_ref
```

Comment:
Note: this is for debug purpose, should be removed

Reply:
## Github context value based on event
Assume `automation/test-docs-build-1` is head branch
`automation/rebuild-docs-manifest` is based branch

|                          |    pull_request_target          | pull_request           |
| ---------------------------- | ------------------------------------------ |----------- |
| **github.head_ref**          | automation/test-docs-build-1            | automation/test-docs-build-1            |
| **github.ref**               | refs/heads/automation/rebuild-docs-manifest | refs/pull/11/merge                   |
| **github.event.pull_request.head.ref** | automation/test-docs-build-1            | automation/test-docs-build-1            |
| **github.event.pull_request.base.ref** | automation/rebuild-docs-manifest         | automation/rebuild-docs-manifest         |